sqlglot/parser.py

```diff
@@ -126,6 +126,7 @@ class Parser:
         TokenType.CONSTRAINT,
         TokenType.DEFAULT,
         TokenType.DELETE,
+        TokenType.EXECUTE,
```

Comment:
doesn't look like you've added enough tokens here, you have a lot more like deterministic and stuff

Reply:
This is the 3rd or so time I've made this mistake. Is there a way to catch this with tests automatically?

Reply:
no because this list is hand-curated, you don't want every token to automatically be a candidate alias. for example

select 1 FROM is not a valid alias

but now that i'm typig it out, maybe we actually need an inverted list. a deny list instead of an allow list...

Reply:
i'll probably do this after your PR. good suggestion

Reply:
hm well, the allow list is currently smaller than the deny list would be

Reply:
i just tried this and it's way too hard. allow list is easier